### ISSUE TITLE
When you join a house, it errors. 

### ISSUE THREAD

**Issue by real2two** [hivenapp/hiven.js#27]
**Please describe the bug or issue in as much detail as possible:**
I've been trying to join a house with Hiven using `const testJoin = client.houses.Join(roomID);`, but it kept erroring:
`(node:9452) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'house' of undefined`

**Please include a reproducible code sample here, if possible:**
```js

client.houses.Join("Valid room ID.");

```

**Further details:**
- Installed hiven.js version: ^2.0.11
- Installed Node.js version: v14.17.0
- Operating system: Windows 10

**Fix**

Since the code at https://github.com/hivenapp/hiven.js/blob/b4ab1185e98f8d9ef5ead562160680b432f1d85a/lib/Collections/House.ts#L91 is now different than the current table version, I don't know if this was already fixed.

I found a way to fix it by changing:
```
            return yield this.collect(getInvite.data.data.house.id, {
                id: getInvite.data.data.house.id,
                name: getInvite.data.data.house.name,
                owner: (_a = this.client) === null || _a === void 0 ? void 0 : _a.users.resolve(getInvite.data.data.house.owner_id),
                icon: getInvite.data.data.house.icon,
                members: getInvite.data.data.house.members,
                rooms: getInvite.data.data.house.rooms
            });
```
to
```
            return yield this.collect(getInvite.data.house.id, {
                id: getInvite.data.house.id,
                name: getInvite.data.house.name,
                owner: (_a = this.client) === null || _a === void 0 ? void 0 : _a.users.resolve(getInvite.data.house.owner_id),
                icon: getInvite.data.house.icon,
                members: getInvite.data.house.members,
                rooms: getInvite.data.house.rooms
            });
```
(.data.data -> .data)

**Comment by callumok2004** [comment]
If you believe you found a solution, you can always change this to a pull request with the fixes you believe will work and it can be reviewed and merged tnen!

**Comment by Looskie** [comment]
Hey! Thanks for the issue, would you like to get the commits for this? If not, no problem, ill just commit it myself

**Comment by real2two** [comment]
I guess, I'll make a fork and a PR.

**Comment by real2two** [comment]
The code seems a bit different on master than using `npm i`, so I hope the changes I make work.

**Comment by real2two** [comment]
https://github.com/hivenapp/hiven.js/pull/31/

**Comment by looskie** [comment]
Merged #31
